### DISCUSSION TITLE
Daily Standup (Monday, May 30th)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#77] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
 ```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Started Signup view

**What are your priorities for the next 24 hours?**:
 - Keep working on it

**Blockers**:
 - Not sure yet how to manage information between phases of the signup

**Shoutouts**:
 - @antonieto looking forward to see you

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Merged #74 
 - Solved history on #76 
 - Added tests for frontend

**What are your priorities for the next 24 hours?**:
 - Try to add tests on backend
 - Signup views

**Blockers**:
 - None

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Kind of finished the proporsal for the aspirant search main view.
 - Started the structure of said view on react/mui (the form)

**What are your priorities for the next 24 hours?**:
 - Add hooks and overall client interaction

**Blockers**:
 - None
 
**Shoutouts**:
 - @CarlosCRG19 for the carry with the testing activity.

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Finished skill model

**What are your priorities for the next 24 hours?**:
 -  Talk with @antonieto  about the new endpoint for searching aspirants by skills

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19  for helping us with the testing activity
